kernelci/api/helper.py

```diff
@@ -111,6 +112,15 @@ def create_job_node(self, job_config, input_node):
             'artifacts': input_node['artifacts'],
             'revision': input_node['revision'],
         }
+        # This information is highly useful, it might be:
+        # lab-name, device-name
+        # kubernetes cluster name
```

Comment:
Just to reduce the number of lines, these two lines can be combined into a single one.

Reply:
Updated